### PR TITLE
feat: default language to match browsers set preference

### PR DESCRIPTION
**Pull request by LPX55**
## Description

- Quick and dirty solution until product team is ready with a language select component. 
- Defaults back to EN if no translation is available for the locale.
- Probably not the best implementation, but it works and its temporary

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

QmbotWg53Pn9Eh1ooLzXxSXRQgLCUrMDMhc6mp1PecvhPj

### CODE REVIEW

**Comment by LPX55** [comment]
Errr yeah, build will obviously fail without the newly PR'ed locales are merged.

File: src/assets/translations/index.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
 import en from './messages-en.json'
+import es from './messages-es.json'
+import fr from './messages-fr.json'
+// import ko from './messages-ko.json' KO locale almost ready
+import pt from './messages-pt.json'
+import ru from './messages-ru.json'
+import zh from './messages-ru.json'
```
Comment (0xdef1cafe) [inline src/assets/translations/index.ts:7]:
```suggestion
import zh from './messages-zh.json'
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

**Review by LPX55 — COMMENTED** [review]
Typo fixed, KO locale removed for next release.

**Review by 0xdef1cafe — APPROVED** [review]

**Review by gomesalexandre — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
fleek is failing because we hadn't merged spanish, CI passes, i'm merging

**Pull request merged**